Packs/GitLab/Integrations/GitLabEventCollector/GitLabEventCollector.py

```diff
@@ -1,179 +1,285 @@
-from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
-from SiemApiModule import *  # noqa # pylint: disable=unused-wildcard-import
+import demistomock as demisto
+from CommonServerPython import *
 
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 VENDOR = "gitlab"
 PRODUCT = "gitlab"
+DEFAULT_LIMIT = 1000
+
+
+''' HELPER FUNCTIONS '''
+
+
+def arg_to_strtime(value: Any) -> Optional[str]:
+    if datetime_obj := arg_to_datetime(value):
+        return datetime_obj.strftime(DATE_FORMAT)
+    return None
+
+
+def prepare_query_params(params: dict, last_run: dict = {}) -> str:
+    """
+    Parses the given inputs into GitLab Events API expected query params URL.
+    """
+    if next_url := last_run.get("next_url"):
+        return next_url
+
+    return f"pagination=keyset&created_after={arg_to_strtime(params.get('after'))}&per_page=100"
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def get_events_request(self, next_url_link: str) -> tuple:
+        """
+        Sends request to get events from GitLab.
+        """
+        response = self._http_request(method='GET', url_suffix=f'?{next_url_link}', resp_type='response')
```

Comment:
```suggestion
    def get_events_request(self, url_suffix: str, query_params: str) -> tuple:
        """
        Sends request to get events from GitLab.
        """
        response = self._http_request(method='GET', url_suffix=f'{url_suffix}?{query_params}', resp_type='response')
```
added url_suffix as suggested in the previous comment, lmk if I am missing somthing